MobiusCore/Source/EffectHandlers/EffectHandler.swift

```diff
@@ -25,8 +25,10 @@
 ///
 /// Note: `EffectHandler` should be used in conjunction with an `EffectRouter`.
 public protocol EffectHandler {
-    associatedtype Effect
-    associatedtype Event
+    associatedtype Input
+    associatedtype Output
+    typealias Effect = Input
```

Comment:
We can keep these names around to avoid breaking existing consumers.

Reply:
👍 Amazing!